_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -1378,6 +1378,26 @@ bool MfxHwH264Encode::IsCmNeededForSCD(
     return useCm;
 }
 
+bool MfxHwH264Encode::IsMctfSupported(
+    MfxVideoParam const & video)
+{
+    bool
+        isSupported = false;
```

Comment:
nit: you don't actually need additional variable, it just complicates logic, just return values directly instead of assigning it to variable

Reply:
I think it makes it easier to debug but I will follow your recommendation